packages/openapi-spec-builder/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@loopback/openapi-spec-builder",
+  "version": "4.0.0-alpha.1",
+  "description": "Builder API making it easy to create OpenAPI Spec/Swagger documents in your tests",
```

Comment:
Making it easy to create OpenAPI (Swagger) specification documents in your tests using the builder pattern.